marklogic-data-hub/src/test/java/com/marklogic/hub/searchfacets/SearchFacetsTest.java

```diff
@@ -11,17 +11,19 @@
 import com.marklogic.client.query.StructuredQueryBuilder;
 import com.marklogic.client.query.StructuredQueryDefinition;
 import com.marklogic.hub.ApplicationConfig;
+import com.marklogic.hub.DataHub;
+import com.marklogic.hub.EntityManager;
 import com.marklogic.hub.HubConfig;
 import com.marklogic.hub.HubTestBase;
 import com.marklogic.hub.flow.impl.FlowRunnerImpl;
 import com.marklogic.hub.impl.HubConfigImpl;
 import com.marklogic.hub.step.StepDefinition;
 import org.apache.commons.io.FileUtils;
 import org.custommonkey.xmlunit.XMLUnit;
+import org.junit.Test;
```

Comment:
This needs to be org.junit.jupiter.api.Test.   I ran the test locally, and both failed with a null-pointer on the "hubConfig" object because it wasn't initialized.

Also, after I fixed that locally, I ran the tests again but I got the following error:

om.marklogic.client.FailedRequestException: Local message: search failed: Bad Request. Server Message: XDMP-PATHRIDXNOTFOUND: cts:path-reference("//*:instance/e2eentity/name", ("type=string", "collation=http://marklogic.com/collation/codepoint")) -- No string path range index for //*:instance/e2eentity/name collation=http://marklogic.com/collation/codepoint